samples/tab-navbar-menu/ts/src/index.css

```diff
@@ -0,0 +1,8 @@
+body {
+  margin: 5em;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Do we need this ?

Reply:
This is the standard CSS. We have created a common style that may be a feature.